frontend/app/forms/proof-of-completed-service/proof-of-completed-service.web.component.js

```diff
@@ -161,7 +161,107 @@ export default function ProofOfCompletedService_Web({ data }) {
           <small>(Utah Code 78B-7-101 et seq.)</small>.
         </p>
       </Section>
-      <Section name="4. Service by Third Person (continued)"></Section>
+      <Section name="4. Service by Third Person (continued)">
+        <Checkbox
+          dataKey="case.proofOfCompletedServiceDeliveredForm"
+          label="On"
+        />
+        <TextInput
+          dataKey="person.proofOfCompletedServiceDelieveredDate"
+          label="date MM-DD-YYYY"
+        />
+        <p>I went to</p>
+        <TextInput
+          dataKey="person.proofOfCompletedServiceDelieveredAddressZip"
+          label="(address)"
+        />
+        <p>and I delivered the documents listed in paragraph 1 to</p>
+        <TextInput
+          dataKey="person.proofOfCompletedServiceDelieveredName"
+          label="(name)"
+        />
+        <p>
+          who is <small>(Check one.)</small>
+        </p>
+        <Checkbox
+          dataKey="person.defendant"
+          label="the named defendant/respondent"
+        />
+        <Checkbox
+          dataKey="person.plaintiff"
+          label="the named plaintiff/petitioner"
+        />
+        <Checkbox
+          dataKey="person.proofOfCompletedServicePersonSuitableAge"
+          label="a person of suitable age and discretion residing at that address, which is the named party's residence"
+        />
+        {data.person.proofOfCompletedServicePersonSuitableAge && (
+          <TextArea
+            dataKey="person.proofOfCompletedServicePersonSuitableAgeDescribe"
+            label="Describe why the person lives at the named party's residence and why they are of suitable age and discretion"
+          />
+        )}
+
+        <Checkbox
+          dataKey="person.agent"
+          label="an agent authorized by appointment or by law to receive servie of process on behalf of the named party"
+        />
+        <br></br>
+        <small>
+          If serving a corportion, partnership, or an unincorporated association
+        </small>
+        <Checkbox
+          dataKey="person.officer"
+          label="an officer, a managing agent, general agent"
+        />
+        <br></br>
+        <strong>OR</strong>
+        <Checkbox
+          dataKey="person.authorizedAgent"
+          label="an agent authorized by appointment or by law to receive service of
+          process and by also mailing a copy of the complaint and summons to the
+          named party, if the agent is one authorized by statute to receive process
+          and the statute so requires."
+        />
+        <br></br>
+        <small>If serving a city or town</small>
+        <Checkbox dataKey="person.recorder" label="the city/town recorder" />
+        <br></br>
+        <small>If serving a county</small>
+        <Checkbox dataKey="person.clerk" label="the county clerk" />
+        <br></br>
+        <small>If serving the state</small>
+        <Checkbox
+          dataKey="person.attorneyGeneral"
```

Comment:
If I'm reading the form right, this is not actually the name of the attorney general. It's the name of a person who was served the document **in addition to the attorney general**. Additionally, this should be on `case`, not person, since the served documents are specific to this case.

Reply:
I believe lines 178-184 is the individual the documents were served to followed by the designation "who is..." And this box would indicate that person is the attorney general. The text field after this in line 237 would be an add'l person required by the statute.

Reply:
Ah you are completely right. I thought this was a TextInput, not a Checkbox. We definitely need the checkbox for this. Disregard what I said :)